Presentation.Web/Swagger/SwashbuckleExtensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Swashbuckle.Swagger;
+using System.Collections.Generic;
+
+namespace Presentation.Web.Swagger
+{
+    public static class SwashbuckleExtensions
+    {
+        public static IEnumerable<Operation> EnumerateOperations(this PathItem pathItem)
+        {
+            if (pathItem == null)
+            {
+                yield break;
+            }
+            yield return pathItem.get;
+            yield return pathItem.post;
+            yield return pathItem.put;
+            yield return pathItem.patch;
+            yield return pathItem.delete;
+            yield return pathItem.options;
+            yield return pathItem.head;
+        }
+
+        public static IEnumerable<Schema> EnumerateSchema(this Operation operation)
+        {
+            if (operation == null)
+            {
+                yield break;
+            }
+            //Response schemas
+            foreach (var response in operation.responses ?? new Dictionary<string, Response>())
+            {
+                yield return response.Value.schema;
+                if (response.Value.schema?.items != null)
+                {
+                    yield return response.Value.schema.items;
+                }
+            }
+            //Parameter schemas
+            foreach (var parameter in operation.parameters ?? new List<Parameter>())
+            {
+                yield return parameter.schema;
+            }
+        }
+        
+        public static IEnumerable<Schema> StartSchemaEnumeration(this Schema schema, IDictionary<string, Schema> listOfDefinition)
```

Comment:
`listOfDefinition`??? odd name for a dictionary.. why not `schemaByTypeName` which is what it is?